Makefile

```diff
@@ -95,12 +95,22 @@ else
 	echo "Skipping publish for branch: $(BRANCH), artifacts are published only from master branch"
 endif
 
-gen-yaml:
+download-kustomize:
+	curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
+	grep browser_download |\
+  	grep $(OPSYS) |\
+  	cut -d '"' -f 4 |\
+  	xargs curl -O -L
+	mv kustomize_kustomize\.v*_$(OPSYS)_amd64 kustomize
+	chmod u+x kustomize
+
+
+gen-yaml: download-kustomize
 	mkdir -p ./out/yaml
 	mkdir -p ./out/scripts
-	kustomize build ./install/admiral/overlays/demosinglecluster/ > ./out/yaml/demosinglecluster.yaml
-	kustomize build ./install/admiralremote/base/ > ./out/yaml/remotecluster.yaml
-	kustomize build ./install/sample/base/ > ./out/yaml/sample.yaml
+	./kustomize build ./install/admiral/overlays/demosinglecluster/ > ./out/yaml/demosinglecluster.yaml
```

Comment:
This might not work when running this locally. Do you want to copy kustomize to `$PATH` in download-kustomize, that will probably help solve this.

Reply:
Agree ! Modifying it.